packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -441,7 +485,7 @@ class FlutterPlugin implements Plugin<Project> {
         pluginProject.afterEvaluate {
             // Checks if there is a mismatch between the plugin compileSdkVersion and the project compileSdkVersion.
             if (pluginProject.android.compileSdkVersion > project.android.compileSdkVersion) {
-                project.logger.quiet("Warning: The plugin ${pluginName} requires Android SDK version ${pluginProject.android.compileSdkVersion.substring(8)}.")
+                project.logger.quiet("Warning: The plugin ${pluginObject.name} requires Android SDK version ${pluginProject.android.compileSdkVersion.substring(8)}.")
```

Comment:
FYI I think compileSdkVersion is deprecated. Also unrelated to your change this error feels like it is missing words like maybe it should say "Plugin xyz requires a lower android sdk version"

Reply:
Moved to #140043